client/src/Components/Login/login.js

```diff
@@ -0,0 +1,116 @@
+import './login.css';
+import { FaFacebookF } from "react-icons/fa"
+import { FaGithub } from "react-icons/fa"
+import { FaGoogle } from "react-icons/fa"
+import { FaLinkedin } from "react-icons/fa"
+
+
+function login() {
+
+    const signUp = () => {
+        document.querySelector(".container_login").classList.add("sign-up-mode");
+        console.log('signUp')
+    };
+
+    const signIn = () => {
+        document.querySelector(".container_login").classList.remove("sign-up-mode");
+    };
+
+    return (
+        <div className="container_login">
+            <div className="forms-container">
+                <div className="signin-signup">
+                    <form action="#" className="sign-in-form">
+                        <h2 className="title">Sign in</h2>
+                        <div className="input-field">
+                            <i className="fas fa-user"></i>
+                            <input type="text" placeholder="Username" />
+                        </div>
+                        <div className="input-field">
+                            <i className="fas fa-lock"></i>
+                            <input type="password" placeholder="Password" />
+                        </div>
+                        <input type="submit" value="Login" className="btn solid" />
+                        <p className="social-text">Or Sign in with social platforms</p>
+                        <div className="social-media">
+                            <a href="#" className="social-icon">
```

Comment:
In React, relative URLs should always be handled by the link tag provided by the React Router, and pure anchor tags should only be used for absolute paths.

Reply:
Thanks Neel I know I have used anchor tag but it was to give an idea of where I have to apply react routes. I have implemented Routes now thus also have replaced these anchor with routes as suggested. In next PR you will be able to see the changes made